lib/Doctrine/DBAL/Schema/AbstractAsset.php

```diff
@@ -64,9 +64,9 @@ protected function _setName($name)
             $name = $this->trimQuotes($name);
         }
         if (strpos($name, ".") !== false) {
```

Comment:
It could be simplified as:
```php
if (false !== $pos = strrpos($name, '.')) {
    $this->_namespace = substr($name, 0, $pos);
    $name = substr($name, $pos + 1);
}
```
saving one call to `strpos()`.